src/frontend/components/SøknadsSteg/Oppsummering/Oppsummering.tsx

```diff
@@ -63,25 +85,220 @@ const Oppsummering: React.FC = () => {
                     tittel={<SpråkTekst id={'omdeg.opphold-i-norge.spm'} />}
                     søknadsvar={søknad.søker.oppholderSegINorge.svar}
                 />
-                {søknad.søker.oppholdsland.svar !== undefined && (
+                {søknad.søker.oppholdsland.svar && (
+                    <OppsummeringFelt
+                        tittel={<SpråkTekst id={'omdeg.opphold-i-norge.land.spm'} />}
+                        søknadsvar={landkodeTilSpråk(
+                            søknad.søker.oppholdsland.svar,
+                            intl.defaultLocale
+                        )}
+                    />
+                )}
+                {søknad.søker.oppholdslandDato.svar && (
+                    <OppsummeringFelt
+                        tittel={<SpråkTekst id={'omdeg.opphold-i-norge.dato.spm'} />}
+                        søknadsvar={søknad.søker.oppholdslandDato.svar}
+                    />
+                )}
+
+                <OppsummeringFelt
+                    tittel={<SpråkTekst id={'omdeg.opphold-sammenhengende.spm'} />}
+                    søknadsvar={søknad.søker.værtINorgeITolvMåneder.svar}
+                />
+                {søknad.søker.komTilNorgeDato.svar && (
                     <>
                         <OppsummeringFelt
-                            tittel={<SpråkTekst id={'omdeg.opphold-i-norge.land.spm'} />}
-                            søknadsvar={landkodeTilSpråk(
-                                søknad.søker.oppholdsland.svar,
-                                intl.locale
-                            )}
+                            tittel={<SpråkTekst id={'omdeg.opphold-sammenhengende.dato.spm'} />}
+                            søknadsvar={søknad.søker.komTilNorgeDato.svar}
                         />
                         <OppsummeringFelt
-                            tittel={<SpråkTekst id={'omdeg.opphold-i-norge.dato.spm'} />}
-                            søknadsvar={søknad.søker.oppholdslandDato.svar}
+                            tittel={<SpråkTekst id={'omdeg.planlagt-opphold-sammenhengende.spm'} />}
+                            søknadsvar={søknad.søker.planleggerÅBoINorgeTolvMnd.svar}
```

Comment:
Lage en egen sjekk for denne i stedet for komTilNorgeDato? :)

Reply:
Ja, sant! Endrer nå